documentation/modules/cruise-control/ref-cruise-control-configuration.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// assembly-cruise-control-concepts.adoc
+
+[id='ref-cruise-control-configuration-{context}']
+= Cruise Control configuration
+
+The `config` property in `Kafka.spec.cruiseControl` contains Cruise Control configuration options as keys with values in one of the following JSON types:
```

Comment:
Here we are referring to "configuration options", instead of "properties". I think we need to consistent in what we call these things.

Reply:
FWIW I think using configuration options throughout would be better. But I may just be being needlessly picky.

Reply:
`Kafka.spec.cruiseControl.config` is a property that holds the the configuration options for Cruise Control. `Kafka.spec.cruiseControl` is a property that has other properties for configuring the Cruise Control deployment